trlc/parser.py

```diff
@@ -1902,44 +1902,6 @@ def parse_rsl_file(self):
 
         return ok
 
-    def parse_check_file(self):
-        # lobster-trace: LRM.Check_File
```

Comment:
If you remove traces to the LRM
Always check if the corresponding content needs to be removed too